aiohttp/web_reqrep.py

```diff
@@ -321,6 +321,15 @@ def json(self, *, loads=json.loads, loader=None):
         return loads(body)
 
     @asyncio.coroutine
+    def multipart(self, *, reader=multipart.MultipartReader):
+        """Return async iterator to process BODY as multipart."""
+        if not self.content_type.startswith('multipart/'):
+            warnings.warn('Attempt to process multipart request with'
```

Comment:
For `application/x-www-form-urlencoded` I think we can have just `.form()` which returns `MultiDictProxy(MultiDict())` as like `.post()` and can be called multiple times.